modules/splitter/monitoring-the-solver-proc.adoc

```diff
@@ -134,3 +134,46 @@ of the measuring.
 * `optaplanner.solver.solve-length.seconds-max`: run time of the
 longest-running currently active solver.
 * `optaplanner.solver.solve-length.seconds-duration-sum`: the sum of each active solver's solve duration. For example, if there are two active solvers, one running for three minutes and the other for one minute, the total solve time is four minutes.
+
+== Additional Metrics
+
+For more detailed monitoring, you can configure {PLANNER_SHORT} to monitor additional metrics at a performance cost. The following example uses the `BEST_SCORE` and `SCORE_CALCULATION_COUNT` metric:
+
+[source,xml,options="nowrap"]
+----
+<solver xmlns="https://www.optaplanner.org/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://www.optaplanner.org/xsd/solver https://www.optaplanner.org/xsd/solver/solver.xsd">
+  <monitoring>
+    <metric>BEST_SCORE</metric>
+    <metric>SCORE_CALCULATION_COUNT</metric>
+    ...
+  </monitoring>
+  ...
+</solver>
+----
+
+You can enable the following metrics this the configuration:
+
+* `SOLVE_DURATION` (default, Micrometer meter ID: "optaplanner.solver.solve.duration"): Measures the duration of solving for the longest active solver, the number of active solvers, and the cumulative duration of all active solvers.
```

Comment:
"default" does not sound clear to me, maybe "enabled by default"? (applies to a few others as well)

Reply:
Also should we perhaps use ` formatting instead of " quotes for the Micrometer meter ID? Applies to all metrics.